app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -179,16 +193,17 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 	)
 	private val drivingAdvancedFields: List<Flow<String>> = listOf(
 			drivingMode, drivingGearLabel,
-			accelContact, brakeState,
```

Comment:
Understood! Some cars don't show some of these fields though. I presume RPMs are only in sporty models, and the person who contributed the brake and clutch values came up with values based on their car's behavior.
These fields are in the Advanced view specifically because they are not as commonly useful and are uncertain (which is why I haven't added them to be translated).

Reply:
BMWs are in common sporty cars ;-) and I haven't seen any BMW without RPM in Dashboard by default...

Reply:
I'll make another solution for this: reverting "advanced data view" to your original setting and adding additional view "Performance data" with my view.

Reply:
New version committed to Github
List of available screens (advanced view)
![20230610_085646](https://github.com/BimmerGestalt/AAIdrive/assets/81849700/6c4d8c1d-5e15-437f-a2b0-7c9e4a17641a)
driving details (reverted to original)
![20230610_085728](https://github.com/BimmerGestalt/AAIdrive/assets/81849700/826b9271-0bcf-4c6b-a5ce-ecaaae3f0e96)
Performance Details:
![20230610_085740](https://github.com/BimmerGestalt/AAIdrive/assets/81849700/11156976-8986-4713-a8a5-6718771a433a)